standard/expressions.md

```diff
@@ -3575,6 +3652,26 @@ The result of evaluating `~x`, where `X` is an expression of an enumeration ty
 
 Lifted ([§12.4.8](expressions.md#1248-lifted-operators)) forms of the unlifted predefined bitwise complement operators defined above are also predefined.
 
+### §hat-operator Hat/index from-end operator
+
+The unary `^` operator is called the *hat* (or *index from-end*) operator. The predefined hat operator is:
+
+```csharp
+Index operator ^(int x);
```

Comment:
For the types `Index` and `Range`, unless we make a general statement somewhere that unadorned uses of these names mean `System.Index` and `System.Range`, we should explicitly qualify them. While my proposal chose the latter, I'm happy to go the former route instead. If we agree on the latter, I'll draft some words and figure out where to put them.

Reply:
The types are added to §C.5 which I think covers it. Also the first use of the names in their respective descriptive clauses (§24.2 & §24.3) includes `System.` as I thought spelling the out would be helpful, also consistent with §C.5 I think but maybe not consistent with the rest of §24?